backend/main.go

```diff
@@ -81,6 +84,8 @@ func killServer() {
 }
 
 func main() {
+	config.LoadConfig()
```

Comment:
Load data from environment variables into `config` and exit if we got any missing env vars.